lib/index.js

```diff
@@ -781,6 +781,43 @@ class GalleryVideoArticle extends BaseAsset {
 
 exports.GalleryVideoArticle = GalleryVideoArticle;
 
+/**
+ * A simple wrapper over VideoAsset that should be used for toplevel videos.
+ *
+ * This asset type is for videos that aren't embedded as part of another
+ * piece of content, but should instead be directly accessible from
+ * content listings and search results. For instance, if you were building
+ * an app where the only content was videos and those videos required no
+ * other context other than some displayable metadata such as the
+ * authors and a brief synopsis, then you should prefer to use this asset type.
+ *
+ * In the past, developers used GalleryVideoArticle, which wrapped
+ * the video in another HTML page. That asset type is now deprecated
+ * since apps are capable of displaying the video and their corresponding
```

Comment:
`displaying the video and their` -> `displaying videos and their` or `displaying the video and its`